dist/src/context.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
```

Comment:
I think we shouldn't include built files in git repo. Usually we do have dist folder specified in gitignore file.

Reply:
okay noted.

Reply:
Added dist in the .gitignore